examples/README.md

```diff
@@ -0,0 +1,75 @@
+
+## A) Examples of high-level analysis (pipelines) using TIAToolbox
+List of Jupyter notebooks that demonstrate how you can use TIAToolbox to simplify high-level analysis in computational pathology.
+
+### 1- Prediction of molecular pathways and key mutations in colorectal cancer from whole slide images
+Prediction of molecular pathways and key mutations directly from Haematoxylin and Eosin stained histology images can help bypass additional genetic (e.g., polymerase chain reaction or PCR) or immunohistochemistry (IHC) testing, which can therefore save both money and time. In this example notebook, we show how you can use TIAToolbox's pretrained models to do reproduce the inference results obtained by IDaRS pipeline introduced in [Bilal et al](https://bit.ly/3IwL6vv). In TIAToolbox, we include models that are capable of predicting the following entities in whole slide images:
+- Microsatellite instability (MSI)
+- Hypermutation density
+- Chromosomal instability
+- CpG island methylator phenotype (CIMP)-high prediction
+- BRAF mutation 
+- TP53 mutation
+
+![image](https://user-images.githubusercontent.com/74412979/145237655-a6361f26-3861-467f-b2f7-1f2c44e3b1a2.png)
+
+### 2- Prediction of HER2 status in breast cancer from H&E stained whole slide images
+This example notebook demonstrates how the functionalities available in TIAToolbox can be used to reproduce the ["SlideGraph+ method" ("SlideGraph+: Whole Slide Image-Level Graphs to Predict HER2Status in Breast Cancer" by Lu et al. (2021))](https://arxiv.org/abs/2110.06042) to predict HER2 status of breast cancer samples from H&E stained whole slide images. As a brief overview, this method involves several steps to generating a graph that represents a whole slide image (WSI) and then directly feeding it into a special convolutional graph network, called SlideGraph, to predict whether that WSI is HER2 negative or positive.
+
+![image](https://user-images.githubusercontent.com/74412979/145244421-ad2f28fe-1361-44b8-a82f-707fd72b0a28.png)
+
+
+## B) Examples of TIAToolbox functionalities
+List of Jupyter notebooks that are dedicated to explaining the essential functionalities of TIAToolbox.
+
+### 1- Reading Whole Slide Images ([01-wsi-reading.ipynb](https://github.com/TissueImageAnalytics/tiatoolbox/blob/master/examples/01-wsi-reading.ipynb))
+This notebook shows you how you can use TIAToolbox to read different kinds of WSIs. TIAToolbox supports various WSI formats acquired by different sources a list of which is available [here](https://tia-toolbox.readthedocs.io/en/latest/usage.html?highlight=wsiread#tiatoolbox.wsicore.wsireader.get_wsireader). In this example, you will also learn about a couple of well-known techniques for WSI mask and patch extraction.
+
+![image](https://user-images.githubusercontent.com/74412979/145223963-f5cc3efc-5762-43c1-b040-c1f738a98e1b.png) ![image](https://user-images.githubusercontent.com/74412979/145224002-b61eb074-5b55-45c9-a45c-9b527437be2c.png)
```

Comment:
The text on this image doesn't appear good in dark mode. @John-P  please can you help to replace this image so it is suited to both normal ("bright") and dark mode.

Reply:
corrected in commit eebdd8b